feature-libs/cart/base/core/facade/active-cart.service.ts

```diff
@@ -527,6 +527,26 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
       .subscribe();
   }
 
+  /**
+   * Reloads current active cart
+   */
+  reloadCurrentActiveCart() {
+    const cartId = OCC_CART_ID_CURRENT;
+    this.userIdService
+      .takeUserId()
+      .pipe(
+        take(1),
+        map((userId) => {
+          this.multiCartFacade.loadCart({
+            cartId,
+            userId,
+            extraData: { active: true },
+          });
+        })
+      )
+      .subscribe();
+  }
+
```

Comment:
This method lacks unit test coverage and facade exposure.

However, it seems at first glance that this function is maybe not required.  Could this logic can be accomplished by existing facade service functions, like perhaps `MultiCartFacade.reloadCart`?  What do you think?

Reply:
It does end up calling `MultiCartFacade.reloadCart`. The component requesting reload doesn't have a cartId so a method was needed to pass the "current" to reloadCart